rust/src/wallet.rs

```diff
@@ -228,6 +230,41 @@ pub fn get_bitcoin_tx_history() -> Result<Vec<bdk::TransactionDetails>> {
     get_wallet()?.get_bitcoin_tx_history()
 }
 
+pub fn get_lightning_history() -> Result<Vec<LightningTransaction>> {
+    let (outbound, inbound) = {
+        let lightning = &get_wallet()?.lightning;
+        let x = (
+            lightning.outbound_payments.lock().unwrap().clone(),
+            lightning.inbound_payments.lock().unwrap().clone(),
+        );
+        x
+    };
+    let mut outbound = outbound
+        .iter()
+        .map(|(_, payment_info)| LightningTransaction {
+            tx_type: LightningTransactionType::Payment,
+            flow: Flow::Outbound,
+            sats: Amount::from(payment_info.amt_msat.clone()).to_sat(),
+            status: payment_info.status.clone().into(),
+            timestamp: payment_info.timestamp,
+        })
+        .collect();
+
+    let mut inbound = inbound
+        .iter()
+        .map(|(_, payment_info)| LightningTransaction {
+            tx_type: LightningTransactionType::Payment,
+            flow: Flow::Outbound,
```

Comment:
```suggestion
            flow: Flow::Inbound,
```

🛠️ I think this is a copy-paster that should be fixed :D

Reply:
Good catch. Fixed